plugins/modules/subnet.py

```diff
@@ -129,6 +129,7 @@
     description:
       - Remote execution Smart proxies for this subnet
       - This option is only available if the remote_execution plugin is installed.
+      - This will always report changed when used with remote_execution < 4.1.0, due to a bug in plugin.
```

Comment:
```suggestion
      - This will always report I(changed=true) when used with I(remote_execution < 4.1.0), due to a bug in the plugin.
```

Not quite sure if "I" is the right one in the second place.

Reply:
It's as good as any other, as they don't fit anyways :see_no_evil: